File: food-order/src/Domain/OrderItem.cs
namespace food_order.Domain
{
    public class OrderItem
    {
        public long? Id { get; }
        public string Uuid { get; }
        public string Name { get; }
        public int Amount { get; }
        public decimal UnitValue { get; }

        public OrderItem(string uuid, string name, int amount, decimal unitValue) : 
            this(null, uuid, name, amount, unitValue) { }
        
        public OrderItem(long? id, string uuid, string name, int amount, decimal unitValue) {
            this.Id = id;
            this.Uuid = uuid;
            this.Name = name;
            this.Amount = amount;
            this.UnitValue = unitValue;
        }

        OrderItem() { }
    }
}

File: food-review/src/main/java/com/food/review/domain/OrderItem.java
package com.food.review.domain;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

import java.io.Serializable;
import java.math.BigDecimal;
import java.util.UUID;

@Getter
@Setter(AccessLevel.PRIVATE)
@Builder(toBuilder = true)
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@NoArgsConstructor(access = AccessLevel.PRIVATE)
@ToString(exclude = {"id"})
@EqualsAndHashCode
public class OrderItem implements Serializable {

    private static final long serialVersionUID = -4638841846509670495L;

    private UUID id;
    private String name;
    private Integer amount;
    private BigDecimal unitValue;

}
